christine-anthony.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en">
+<html>
+
+  <head>
+    <meta charset="uft-8">
+  <!-- why doesnt the fond change??? -->
```

Comment:
Meta charset does not have anything to do with fonts. See [this intro](https://www.w3.org/International/questions/qa-what-is-encoding).

Reply:
I understand charsets now. I really like the last article by Joel Spolsky linked at the end of the intro about unicode and character sets.